README.md

```diff
@@ -124,9 +124,11 @@ Deploy Station with [Docker](https://www.docker.com/). Please replace
 `FIL_WALLET_ADDRESS`.
 
 ```bash
-$ docker run \
-	-e FIL_WALLET_ADDRESS=f1abjxfbp274xpdqcpuaykwkfb43omjotacm2p3za \
-	-d ghcr.io/filecoin-station/core
+# Run container
+$ docker run --name station -d -e FIL_WALLET_ADDRESS=f1abjxfbp274xpdqcpuaykwkfb43omjotacm2p3za ghcr.io/filecoin-station/core
+
+# Get logs
+$ docker exec station ./bin/station.js logs
```

Comment:
Let's not use shell comments and structure the text in Markdown instead.

GitHub does not seem to support a markdown snippet with embedded fenced code blocks; please replace two backticks with three backticks in the content below.


```md
Deploy Station with [Docker](https://www.docker.com/). Please replace 
`FIL_WALLET_ADDRESS`.

``bash
$ docker run (...)
``

Inspect logs:

``bash
$ docker exec station ./bin/station.js logs
``
```

Reply:
Good idea!